spec/services/pairing_strategies/single_sided_swiss_spec.rb

```diff
@@ -355,26 +355,100 @@
           end
         end
 
+        # This shouldn't be in the second round context - it needs at least 2 completed rounds to test.
+        # TODO(plural): Move this out.
         it 'avoids third matchups' do
```

Comment:
Does this still need doing?

Reply:
yes, but i would rather do a split here separately.  It isn't impacting anything functionally.

Reply:
I had time on a flight today so i went ahead and pulled out a 3rd round context.